sample_sensor_kit_launch/launch/pointcloud_preprocessor.launch.py

```diff
@@ -86,7 +86,8 @@ def add_launch_arg(name: str, default_value=None):
     add_launch_arg("use_multithread", "False")
     add_launch_arg("use_intra_process", "False")
     add_launch_arg("use_pointcloud_container", "False")
-    add_launch_arg("container_name", "pointcloud_preprocessor_container")
+    add_launch_arg("pointcloud_container_name", "pointcloud_container")
```

Comment:
To reflect the change here (`container_name` -> `pointcloud_container_name` ), perhaps https://github.com/autowarefoundation/sample_sensor_kit_launch/blob/8737af12a3c0160ac04b8b4238e3d29570108980/sample_sensor_kit_launch/launch/lidar.launch.xml#L84 should be modified to `pointcloud_container_name`?

Reply:
Right, thank you. Fixed: https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/85/commits/44c503c1f42cdfa44b513d91d708b873ca35c2dc